lua/lazyvim/config/keymaps.lua

```diff
@@ -110,6 +110,7 @@ map("n", "<leader>uf", require("lazyvim.plugins.lsp.format").toggle, { desc = "T
 map("n", "<leader>us", function() Util.toggle("spell") end, { desc = "Toggle Spelling" })
 map("n", "<leader>uw", function() Util.toggle("wrap") end, { desc = "Toggle Word Wrap" })
 map("n", "<leader>ul", function() Util.toggle_number() end, { desc = "Toggle Line Numbers" })
+map("n", "<leader>up", function() Util.toggle_autopairs() end, { desc = "Toggle Auto Pairs" })
```

Comment:
```lua
if Util.has("mini.pairs") then
  map("n", "<leader>up", function() Util.toggle_autopairs() end, { desc = "Toggle Auto Pairs" })
end
```

Think it would be nice to add a check in case the plugin is disabled.